test/cluster.sh

```diff
@@ -49,7 +49,11 @@ function install_knative_serving() {
 
   echo ">> Bringing up Serving"
   # TODO(#2122): Use RELEASE_YAML once we have monitoring e2e.
-  kubectl apply -f "${RELEASE_NO_MON_YAML}"
+  if [[ -z "${RELEASE_YAML_OVERRIDE}" ]]; then
```

Comment:
@adrcunha WDYT? Do you have a better idea?

Reply:
Yes, but I'll refactor this later.